### PR TITLE
More detailed branch delete message

### LINKED ISSUE #22624: Contradictory messages when deleting branch
### Description

When I delete a branch on the Code -> Branch tab, a message pops up that this delete is permanent and cannot be redone.

After confirming I see the image below where I can restore the branch...

grafik

Whether or not this might work it would be better to have a more consistent wording.

By the way thanks for this great software!

### Gitea Version

1.18.3

### Can you reproduce the bug on the Gitea demo site?

Yes

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

Ubuntu 18.04

### How are you running Gitea?

Manual server installation with systemd

### Database

MySQL

### PR DESCRIPTION
**Pull request by kernie**
Fix  #22624 . Message explains that deleted branch can only be restored for a short time period.

### CODE REVIEW

**Comment by zeripath** [comment]
I perhaps this should just be a warning of `this might not be possible to undo` instead?

**Comment by kernie** [comment]
What about
`Deleting a branch is permanent. It can only be undone shortly after the deletion. Continuue?`

**Comment by a1012112796** [comment]
we can load this config value by backend directly, is't it? so I think we can show `Deleting a branch is permanent. It can only be undone in %d days. Continuue?`

**Comment by kernie** [comment]
> 

I like that! It gives exact information without disturbing the user with technical details.

**Comment by wxiaoguang** [comment]
I think it's good to use the most strict warning for end users, do not make users expect some lucky miracle.

For example, what about the `git cleanup task` is scheduled to run in next minute?


So I'd like to keep the old wording. Users should keep in mind they shouldn't expect a deleted branch could be recovered.

**Comment by kernie** [comment]
I support a strict wording! However, I tiny hint that the repo is scheduled for removal helps understand that it does not disappear immediately...

I don't think we need to spend more time for this issue since the gain is minimal. Please accept or reject to your liking.

Thanks

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -2306,7 +2306,7 @@ branch.already_exists = A branch named '%s' already exists.
 branch.delete_head = Delete
 branch.delete = Delete Branch '%s'
 branch.delete_html = Delete Branch
-branch.delete_desc = Deleting a branch is permanent. It <strong>CANNOT</strong> be undone. Continue?
+branch.delete_desc = Deleting a branch is permanent after 'git cleanup task' was run (by default once per day). It CANNOT be undone. Continue?
```
Comment (wxiaoguang) [inline options/locale/locale_en-US.ini]:
```suggestion
branch.delete_desc = Deleting a branch is permanent. Although the deleted branch may exist for a short time before cleaning up, in most cases it CANNOT be undone. Continue?
```

What do you think about this?

Comment (kernie) [reply options/locale/locale_en-US.ini]: Ok for me.

**Review by wxiaoguang — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Pull request merged**